### DISCUSSION TITLE
Vertical Scroll only on sub-Rows

### DISCUSSION THREAD

**Discussion by HussainAhmadDev** [TanStack/table#5780] [Q&A]
how can I show a vertical scroll on sub-rows?
Actually, In my case, I have too many sub-rows I want to maximize their height to let's say 200px

**Comment by KevinVandy** [comment]
You could make a tbody for every group of sub rows. Every root row could be it's own tbody too.

**Reply by HussainAhmadDev** [reply]
> You could make a tbody for every group of sub rows. Every root row could be it's own tbody too.

I didn't understand the thing is all the rows/subrows have common container the tablebody how can I create an separate container for subrows - should I have to filter out the subrows and them put them in Dom ?
it'll be better if you can give me an example

**Reply by moeen-basra** [reply]
I'm able to achieve this with some hack but it works along with react window, still struggling with [tanstack react virtuali](https://tanstack.com/virtual/latest), If you still need help I can share the some code

**Reply by HussainAhmadDev** [reply]
> I'm able to achieve this with some hack but it works along with react window, still struggling with [tanstack react virtuali](https://tanstack.com/virtual/latest), If you still need help I can share the some code

It'll be amazing @moeen-basra - please share the code snippets
